### PR TITLE
Rename directory back to osv-scanner

### PR DESCRIPTION
**Pull request by andrewpollock**
Turns out we want the resultant binary to have "osv" in the name.

Fixes: #603

### CODE REVIEW

**Review by oliverchang — COMMENTED** [review]
I think you need to add the newly named files :)

File: tools/osv-scanner/cmd/main.go [hunk 1]
```diff
@@ -11,8 +11,8 @@ import (
 	"path/filepath"
```
Comment (oliverchang) [inline tools/osv-scanner/cmd/main.go:11]: Rename this to `tools/osv-scanner/main.go` ?

Comment (andrewpollock) [reply tools/osv-scanner/cmd/main.go:11]: Isn't this https://pkg.go.dev/path/filepath ?

**Comment by andrewpollock** [comment]
> I think you need to add the newly named files :)

Yeah got a little bit trigger happy and omitted a `git add` post-rename of the directory. Fixed in 
3e83538

File: tools/osv-scanner/cmd/main.go (hunk 1 above)
Comment (oliverchang) [reply tools/osv-scanner/cmd/main.go:11]: I meant the file itself.

Comment (andrewpollock) [reply tools/osv-scanner/cmd/main.go:11]: Oh, got it, move it out of the cmd directory.

Comment (andrewpollock) [reply tools/osv-scanner/cmd/main.go]: Addressed in 49ea940

**Review by oliverchang — APPROVED** [review]

**Pull request merged**